### DISCUSSION TITLE
Apks From Vivo v27 ( bbk / funtouch os ) android 14

### DISCUSSION THREAD

**Discussion by ghost** [Universal-Debloater-Alliance/universal-android-debloater-next-generation#179] [General]
app manager from fdroid is a good app to inspect apk  
but its pure torture, kudos to the person who makes these lists

here is a gdrive folder containing Documents.7z file which is just a pack of all the apks on my phone

https://drive.google.com/drive/folders/1-DTtYdiehKPv-K5ofJ7sTLHNYJ8E-3tL?usp=sharing

i removed some very common and fat apps by google, like googlequicksearch, gms, messages, files, gm , etc . thus reducing size to only 1 gb from 3
upload done ✔

**Comment by ghost** [comment]
what i know that is safe and you should disable :   
#### google apps  
- android.vending
- android.chrome
- google.android.marvin.takback
- google.android.projection.gearhead
- google.android.as
- google.android.ar.core
- google.android.googlequicksearchbox
- all apps that start with "google.android.apps."
- google.android.contacts ( use com.android.contacts instead , i dont use other apps , because they are not able to write to sim card or are buggy or too old ) 
- google.android.dialer ( use com.android.contacts instead or any foss alternative ) 
- google.android.inputmedthod.latin ( recommend using "unexpected keyboard" from fdroid, make sure to install it beforehand ) 
- google.android.hotwordenrollment
- google.android.healthconnect.controller
 - google.android.accessibility.switchaccess
- google.android.apps.messaging ( use [this](https://f-droid.org/en/packages/com.simplemobiletools.smsmessenger/) instead )
- all packages with  "widget" or "livewallpaper" in their name
#### others  
- all facebook apps
- com.bbk.updater ( if you have this disabled , then only you can disable gms  , otherwise device will boot but will randomly crash the system ui )
- all "com.vivo" apps
- all "bbk" apps 
- all "iqoo" apps
- vos.user.vit
- vos.as.vit
- vivo.card ( this is smart sidebar, disabling it takes effect on reboot , but sometimes NOT 🥴, so better just turn it off in accessibility settings first )
- com.mediatek.ygps  
#### gray zone 
> they can be disabled , and dont cause a bootloop but i dont know what it does exactly or what are the consequences of disabling them. ( someone please tell me )  : 
- "sps" or super process system is usued for feedback data collection , but it never runs by itself
- com.touchscreen.vtouchscreencheck ( obvious name , but still i want more info before i can disable it ) 
- bbk.iqoo.logsystem
- vivo.pem
- functouch.uiengine 
- vivo.vivo3rdalgoservice ( camera does works even when this is disabled )
- vivo.globalanimations 
- vivo.epm
- vivo.fuelsummary
- vivo.cota
- android.compos.payload
- all com.mediatek packages ( with some exceptions , they cause bootloop ) 
- vivo.abe
- com.vivo.aiengine
- all packages with "overlay" in their name

- mediatek.atci.service ( used 220kB up and 1.13 MB down ) 
- com.mediatek ( shows data usage equal to the above )

**Comment by AnonymousWP** [comment]
@Blitz517 Could you take a look at this?

**Comment by Blitz517** [comment]
i will check unlisted apps, if you found any mistake in descriptions and advanced or recommended bootloops then comment it here

**Comment by Blitz517** [comment]
anyway we should ask peoples probably for unlisted apps on android 14 because its very problematic extracting OEM rom from at this year
